package/run.sh

```diff
@@ -28,7 +28,7 @@ handle_error() {
       sleep infinity
   fi
   # Annotate self (pod) to signal "error"
-  if [[ -f "${ERROR_LOG_FILE}" ]]; then
+  if [[ -s "${ERROR_LOG_FILE}" ]]; then
```

Comment:
Can you explain why this change is needed?

Reply:
`-f` checks if the file is a regular file or not and returns true if the file is a regular file (not a directory). So even if the error log file is empty, it will still return true because the file exists. Where as `-s` checks if file is not zero size. So if the error log file is empty, it won't execute the block.